<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="aspire984" name="Default test" tests="3" failures="3" timestamp="7 May 2015 09:57:52 GMT" time="119.291" errors="0">
  <testcase name="loginTest" time="26.151" classname="com.projectname.test.functional.login.LoginTest">
    <failure type="java.lang.AssertionError" message="Timed out after 20 seconds waiting for com.projectname.testutils.seleniumutils.SeleniumWebDriver$1@1bb4fce
Build info: version: &amp;apos;2.33.0&amp;apos;, revision: &amp;apos;4ecaf82108b2a6cc6f006aae81961236eba93358&amp;apos;, time: &amp;apos;2013-05-22 12:00:17&amp;apos;
System info: os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;x86&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;1.7.0_13&amp;apos;
Driver info: driver.version: SeleniumWebDriver">
      <![CDATA[java.lang.AssertionError: Timed out after 20 seconds waiting for com.projectname.testutils.seleniumutils.SeleniumWebDriver$1@1bb4fce
Build info: version: '2.33.0', revision: '4ecaf82108b2a6cc6f006aae81961236eba93358', time: '2013-05-22 12:00:17'
System info: os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.7.0_13'
Driver info: driver.version: SeleniumWebDriver
	at org.junit.Assert.fail(Assert.java:93)
	at com.projectname.testutils.genericutility.ExceptionHandler.takeScreenshot(ExceptionHandler.java:74)
	at com.projectname.testutils.genericutility.ExceptionHandler.<init>(ExceptionHandler.java:41)
	at com.projectname.testutils.seleniumutils.SeleniumWebDriver.waitForElement(SeleniumWebDriver.java:152)
	at com.projectname.testutils.pages.IntranetHomePage.<init>(IntranetHomePage.java:22)
	at com.projectname.testutils.pages.LoginPage.login(LoginPage.java:42)
	at com.projectname.test.functional.login.LoginTest.loginTest(LoginTest.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="26.074" classname="com.projectname.test.functional.login.LoginTest">
    <failure type="java.lang.AssertionError" message="Timed out after 20 seconds waiting for com.projectname.testutils.seleniumutils.SeleniumWebDriver$1@1c61093
Build info: version: &amp;apos;2.33.0&amp;apos;, revision: &amp;apos;4ecaf82108b2a6cc6f006aae81961236eba93358&amp;apos;, time: &amp;apos;2013-05-22 12:00:17&amp;apos;
System info: os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;x86&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;1.7.0_13&amp;apos;
Driver info: driver.version: SeleniumWebDriver">
      <![CDATA[java.lang.AssertionError: Timed out after 20 seconds waiting for com.projectname.testutils.seleniumutils.SeleniumWebDriver$1@1c61093
Build info: version: '2.33.0', revision: '4ecaf82108b2a6cc6f006aae81961236eba93358', time: '2013-05-22 12:00:17'
System info: os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.7.0_13'
Driver info: driver.version: SeleniumWebDriver
	at org.junit.Assert.fail(Assert.java:93)
	at com.projectname.testutils.genericutility.ExceptionHandler.takeScreenshot(ExceptionHandler.java:74)
	at com.projectname.testutils.genericutility.ExceptionHandler.<init>(ExceptionHandler.java:41)
	at com.projectname.testutils.seleniumutils.SeleniumWebDriver.waitForElement(SeleniumWebDriver.java:152)
	at com.projectname.testutils.pages.IntranetHomePage.<init>(IntranetHomePage.java:22)
	at com.projectname.testutils.pages.LoginPage.login(LoginPage.java:42)
	at com.projectname.test.functional.login.LoginTest.loginTest(LoginTest.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1049)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1254)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="26.371" classname="com.projectname.test.functional.login.LoginTest">
    <failure type="java.lang.AssertionError" message="Timed out after 20 seconds waiting for com.projectname.testutils.seleniumutils.SeleniumWebDriver$1@139132f
Build info: version: &amp;apos;2.33.0&amp;apos;, revision: &amp;apos;4ecaf82108b2a6cc6f006aae81961236eba93358&amp;apos;, time: &amp;apos;2013-05-22 12:00:17&amp;apos;
System info: os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;x86&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;1.7.0_13&amp;apos;
Driver info: driver.version: SeleniumWebDriver">
      <![CDATA[java.lang.AssertionError: Timed out after 20 seconds waiting for com.projectname.testutils.seleniumutils.SeleniumWebDriver$1@139132f
Build info: version: '2.33.0', revision: '4ecaf82108b2a6cc6f006aae81961236eba93358', time: '2013-05-22 12:00:17'
System info: os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.7.0_13'
Driver info: driver.version: SeleniumWebDriver
	at org.junit.Assert.fail(Assert.java:93)
	at com.projectname.testutils.genericutility.ExceptionHandler.takeScreenshot(ExceptionHandler.java:74)
	at com.projectname.testutils.genericutility.ExceptionHandler.<init>(ExceptionHandler.java:41)
	at com.projectname.testutils.seleniumutils.SeleniumWebDriver.waitForElement(SeleniumWebDriver.java:152)
	at com.projectname.testutils.pages.IntranetHomePage.<init>(IntranetHomePage.java:22)
	at com.projectname.testutils.pages.LoginPage.login(LoginPage.java:42)
	at com.projectname.test.functional.login.LoginTest.loginTest(LoginTest.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1049)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1254)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Default test -->
